pkg/controller/servicebindingrequest/sbrcontroller_test.go

```diff
@@ -0,0 +1,181 @@
+package servicebindingrequest
+
+import (
+	"testing"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+
+	"github.com/redhat-developer/service-binding-operator/pkg/apis/apps/v1alpha1"
+	"github.com/redhat-developer/service-binding-operator/pkg/log"
+)
+
+func Test_newSBRPredicate(t *testing.T) {
```

Comment:
@isutton please use the same standards for naming tests than the other methods, This would be named as `TestSBRControllerNewSBRPredicate` instead.

Reply:
Solved in cc0d39e35f1f219cfd3f041d482c75186f870309.